.claude/agents/ci-triage.md

```
@@ -0,0 +1,401 @@
+│ Analyze logs  │  │ Any E2E failures?  │
+│ Fix locally   │  └─────────┬──────────┘
+│ Push fix      │            │
+└───────────────┘    ┌───────┴───────┐
+                     ▼               ▼
+                   YES              NO
+                     │               │
+                     ▼               ▼
+            ┌────────────────┐  ┌──────────┐
+            │ Check if flaky │  │ All pass │
+            └───────┬────────┘  │ Done!    │
+                    │           └──────────┘
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)